packages/clerk-js/rspack.config.js

```diff
@@ -447,5 +471,5 @@ module.exports = env => {
   const mode = env.production ? 'production' : 'development';
   const analysis = !!env.analysis;
 
-  return isProduction(mode) ? prodConfig({ mode, env, analysis }) : devConfig({ mode, env });
+  return isProduction(mode) ? prodConfig({ mode, analysis }) : devConfig({ mode, env });
```

Comment:
Why'd we remove the `env` from `prodConfig` here?

Reply:
Because it wasn't being used.

https://github.com/clerk/javascript/pull/4517/files/45f64b38c27f0f90a33bc13fd444ea80df7e9b1c#diff-9e85d524e620fb7e1152ba8370d1110520b43c812724374047d307fdfca4f8dfR312